src/client/actions/InteractionCreate.js

```diff
@@ -37,12 +39,27 @@ class InteractionCreateAction extends Action {
         return;
     }
 
+    const interaction = new InteractionType(client, data);
+
     /**
      * Emitted when an interaction is created.
-     * @event Client#interaction
+     * @event Client#interactionCreate
      * @param {Interaction} interaction The interaction which was created
      */
-    client.emit(Events.INTERACTION_CREATE, new InteractionType(client, data));
+    client.emit(Events.INTERACTION_CREATE, interaction);
+
+    if (!deprecationEmitted && client.listenerCount('interaction')) {
+      deprecationEmitted = true;
+      process.emitWarning('The interaction event is deprecated. Use interactionCreate instead', 'DeprecationWarning');
+
+      /**
+       * Emitted when an interaction is created.
+       * @event Client#interaction
+       * @param {Interaction} interaction The interaction which was created
+       * @deprecated
```

Comment:
Do we have "Use xyz instead" support here?

Reply:
Yes, we do: https://jsdoc.app/tags-deprecated.html

Reply:
The question was about the website, not the JSDoc Tag

Reply:
I have no idea, [MessageEmbed#type](https://discord.js.org/#/docs/main/master/class/MessageEmbed?scrollTo=type) is the only thing deprecated before this PR.

cc: @iCrawl

Reply:
We do not.